tools/main.cpp

```diff
@@ -447,10 +447,11 @@ int main(int argc, char **argv)
         exit(1);
     }
 
-    if ((outformat != "svg") && (outformat != "mei") && (outformat != "midi") && (outformat != "timemap")
-        && (outformat != "humdrum") && (outformat != "hum") && (outformat != "pae") && (outformat != "pb-mei")) {
+    if ((outformat != "svg") && (outformat != "mei") && (outformat != "mei-basic") && (outformat != "midi")
+        && (outformat != "timemap") && (outformat != "humdrum") && (outformat != "hum") && (outformat != "pae")
+        && (outformat != "pb-mei")) {
         std::cerr << "Output format (" << outformat
-                  << ") can only be 'mei', 'pb-mei', 'svg', 'midi', 'humdrum' or 'pae'." << std::endl;
+                  << ") can only be 'mei', 'mei-basic', 'pb-mei', 'svg', 'midi', 'humdrum' or 'pae'." << std::endl;
```

Comment:
To me `basic` seems sufficient. Or change `pb-mei` into `mei-pb`?

Reply:
Changed to `mei-pb` and made `pb-mei` deprecated